src/garage/experiment/offline_meta_test_helper.py

```diff
@@ -0,0 +1,248 @@
+            test_task_sampler=meta_sampler,
+            max_path_length=self.max_path_length,
+            n_test_tasks=meta_sampler.n_tasks,
+            n_exploration_traj=self.adapt_rollout_per_task,
+            n_evaluation_traj=self.test_rollout_per_task,
+            prefix='')
+
+        for itr in itrs:
+            log_filename = os.path.join(meta_train_dir,
+                                        'meta-test-itr_{}.csv'.format(itr))
+            logger.add_output(CsvOutput(log_filename))
+            logger.log('Writing into {}'.format(log_filename))
+
+            runner.restore(meta_train_dir, from_epoch=itr)
+            # pylint: disable=protected-access
+            meta_evaluator.evaluate(runner._algo)
+            tabular.record('Iteration', runner._stats.total_epoch)
+            tabular.record('TotalEnvSteps', runner._stats.total_env_steps)
+            logger.log(tabular)
+            logger.dump_output_type(CsvOutput)
+            logger.remove_output_type(CsvOutput)
+
+    def test_many_folders(self, folders, workers, skip_existing, to_merge,
+                          stride):
+        """Do meta-testing for multiple folders.
+
+        Args:
+            folders (list[str]): Experiment folders where snapshots are saved.
+            workers (int): Number of parallel workers.
+            skip_existing (bool): If True, tested iterations will be skipped.
+            to_merge (bool): If True, merge per-iteration meta-test csv to a
+                single meta-test.csv.
+            stride (int): Stride between two tested iterations. If set to 1,
+                all available iterations will be tested.
+
+        """
+        for meta_train_dir in folders:
+            itrs = Snapshotter.get_available_itrs(meta_train_dir)
+            tested_itrs = self._get_tested_itrs(meta_train_dir)
+
+            if skip_existing:
+                itrs = [itr for itr in itrs if itr not in tested_itrs]
+
+            if stride > 1:
+                itrs = itrs[::stride]
+
+            if workers == 0:
+                self.test_one_folder(meta_train_dir, itrs)
+            else:
+                bite_size = math.ceil(len(itrs) / workers)
+                bites = [
+                    itrs[i * bite_size:(i + 1) * bite_size]
+                    for i in range(workers)
+                ]
+
+                children = []
+                for bite_itrs in bites:
+                    if len(bite_itrs) == 0:
+                        continue
+                    pid = os.fork()
+                    if pid == 0:
+                        # In child process
+                        self.test_one_folder(meta_train_dir, bite_itrs)
+                        sys.exit()
+                    else:
+                        # In parent process
+                        children.append(pid)
+
+                for child in children:
+                    os.waitpid(child, 0)
+
+            if to_merge:
+                self._merge_csv(meta_train_dir, itrs)
```

Comment:
This file should probably be in `scripts/`, and should have a `if __name__ == '__main__'` block.